patches/kaggle_datasets.py

```diff
@@ -5,6 +5,7 @@
 
 class KaggleDatasets:
     GET_GCS_PATH_ENDPOINT = '/requests/CopyDatasetVersionToKnownGcsBucketRequest'
+    TIMEOUT_SECS = 600
```

Comment:
This timeout seems quite long for only getting the GCS path... What does the GET_GCS_PATH_ENDPOINT handler do?

Reply:
This is the endpoint that copies a Dataset to a regional GCS bucket. The timeout it huge, because it takes a while.
There is a functionality that Mark checked in earlier to make the copy more or less async, but it is not yet used. See http://b/148790385. Once we start using the other approach (essentially polling), we will not need to use large timeout for http request at least.

Reply:
Make sense. I thought it was only concatenating strings :)

Reply:
> Makes sense. I thought it was only concatenating strings ;)

Haha, it may seem this way based on the name :-)